staging/src/k8s.io/apiextensions-apiserver/test/integration/subresources_test.go

```diff
@@ -0,0 +1,209 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
nit: 2017 or 2018? Commit is dated 2017 but it's 2018 now.

Reply:
Tough question ;)